tests/pds/resiliency_tests.go

```diff
@@ -0,0 +1,56 @@
+package tests
+
+import (
+	"fmt"
+	"net/http"
+
+	. "github.com/onsi/ginkgo"
+	pdslib "github.com/portworx/torpedo/drivers/pds/lib"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+)
+
+var _ = Describe("{RebootActiveNodeDuringDeployment}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("RebootActiveNodeDuringDeployment", "Reboots a Node onto which a pod is coming up", pdsLabels, 0)
+	})
+
+	It("deploy Dataservices", func() {
+		Step("Deploy Data Services", func() {
+			for _, ds := range params.DataServiceToTest {
+				if ds.Name != postgresql {
+					continue
+				}
+				Step("Start deployment, Reboot a node on which deployment is coming up and validate data service", func() {
+					isDeploymentsDeleted = false
+					// Global Resiliency TC marker
+					pdslib.MarkResiliencyTC(true)
+					// Type of failure that this TC needs to cover
+					failuretype := pdslib.TypeOfFailure{
+						Type: active_node_reboot_during_deployment,
+						Method: func() error {
+							return pdslib.RebootActiveNodeDuringDeployment(params.InfraToTest.Namespace)
+						},
+					}
+					pdslib.DefineFailureType(failuretype)
+					// Deploy and Validate this Data service after injecting the type of failure we want to catch
+					deployment, _, _, err = TriggerDeployDataService(ds, params.InfraToTest.Namespace, tenantID, projectID)
+					log.FailOnError(err, "Error while deploying data services")
+					err = pdslib.InduceFailureAfterWaitingForCondition(deployment, namespace)
```

Comment:
as discussed pass the replicas as params

Reply:
Done